src/azure-cli/azure/cli/command_modules/backup/custom.py

```diff
@@ -541,11 +550,51 @@ def list_recovery_points(cmd, client, resource_group_name, vault_name, item, sta
     if use_secondary_region:
         client = recovery_points_crr_cf(cmd.cli_ctx)
 
-    # Get recovery points
-    recovery_points = client.list(vault_name, resource_group_name, fabric_name, container_uri, item_uri, filter_string)
+    if recommended_for_archive:
+        if is_ready_for_move is False:
+            raise InvalidArgumentValueError(
+                """
+                All the recommended archivable recovery points are by default ready for
+                move. Please provide the correct value for --is-ready-for-move.
+                """)
+
+        client = recovery_points_recommended_cf(cmd.cli_ctx)
+        recovery_points = client.list(vault_name, resource_group_name, fabric_name, container_uri, item_uri)
+
+    else:
+        recovery_points = client.list(vault_name, resource_group_name, fabric_name, container_uri, item_uri,
+                                      filter_string)
+
     paged_recovery_points = _get_list_from_paged_response(recovery_points)
+    common.fetch_tier(paged_recovery_points)
 
-    return paged_recovery_points
+    if use_secondary_region:
+        paged_recovery_points = [item for item in paged_recovery_points if item.properties.recovery_point_tier_details
+                                 is None or (item.properties.recovery_point_tier_details is not None and
+                                             item.tier_type != 'VaultArchive')]
+
+    recovery_point_list = common.check_rp_move_readiness(paged_recovery_points, target_tier, is_ready_for_move)
+    recovery_point_list = common.filter_rp_based_on_tier(recovery_point_list, tier)
+    return recovery_point_list
+
+
+def move_recovery_points(cmd, resource_group_name, vault_name, item_name, rp_name, source_tier,
+                         destination_tier):
+
+    container_uri = _get_protection_container_uri_from_id(item_name.id)
+    item_uri = _get_protected_item_uri_from_id(item_name.id)
+
+    if source_tier not in common.tier_type_map.keys():
+        raise InvalidArgumentValueError('This source tier-type is not accepted by move command at present.')
```

Comment:
If this parameter is not accepted, why register this parameter in `_params.py` for `backup recoverypoint move`?